portfolio/src/main/webapp/script.js

```diff
@@ -15,12 +15,55 @@
 /**
  * Adds a random greeting to the page.
  */
+
+
+ async function showServerHello() {
+  const responseFromServer = await fetch('/hello');
+  //const textFromResponse = await responseFromServer.text();
+  const hello = await responseFromServer.json();
+
+
+
+  const helloContainer = document.getElementById('hello-container');
+  helloContainer.innerHTML='';
+
+
+ 
+  helloContainer.appendChild(
+      createListElement('Movie #1: ' + hello.movie1));
+  helloContainer.appendChild(
+      createListElement('Movie #2: ' + hello.movie2));
+  helloContainer.appendChild(
+      createListElement('Movie #3: ' + hello.movie3));
+
+   
+  // Add it to the page.
+
+
+ }
+
+ function randomMovie(){
+    const randoMovies= [ 'You\'ve Got Mail', 'My Big Fat Greek Wedding','Monster\'s University'];
+    const pick = randoMovies[Math.floor(Math.random() * (randoMovies.length))];
```

Comment:
Nice! Consider extracting the code that generates a random number into its own function, especially since this code is repeated twice (once here, and another at line 66).